app/views/pxe/_pxe_image_type_form.html.haml

```diff
@@ -1,24 +1 @@
-- url = url_for_only_path(:action => 'pxe_image_type_form_field_changed', :id => (@pxe_image_type.id || "new"))
-#form_div
-  = render :partial => "layouts/flash_msg"
-  %h3
-    = _('Basic Information')
-  .form-horizontal
-    .form-group
-      %label.col-md-2.control-label
-        = _('Name')
-      .col-md-8
-        = text_field_tag("name", @edit[:new][:name], :maxlength => ViewHelper::MAX_NAME_LEN,
-                      :class => "form-control",
-                      "data-miq_observe" => {:interval => '.5', :url => url}.to_json)
-        = javascript_tag(javascript_focus('name'))
-    .form-group
-      %label.col-md-2.control-label
-        = _('Type')
-      .col-md-8
-        = select_tag("provision_type",
-                      options_for_select([["<#{_('Any')}>", nil]] + @edit[:prov_types].invert.sort, @edit[:new][:provision_type]),
-                      :class    => "selectpicker")
-    :javascript
-       miqInitSelectPicker();
-       miqSelectPickerEvent("provision_type", "#{url}")
+= react 'PxeImageForm'#, :recordId => @pxe_image_type.try(:id).try(:to_s)
```

Comment:
Why the comment?

Reply:
Also this file is unnecessary as it has only one callsite in `_pxe_image_type_details.html.haml:5` so you can just replace that `render` call with the `react` from here and delete this file completely.